packages/core/less/verticalnavdivider.less

```diff
@@ -0,0 +1,13 @@
+.vertical-sub-header-pf {
+  line-height: 25px;
+  font-size: @font-size-base;
+  margin-left: 20px;
+  padding: 5px 0 2px;
```

Comment:
To get the spacing that @mcarrano is asking for, the value here should be `padding: 20px 0 2px 5px;`